src/_util.js

```diff
@@ -330,7 +330,7 @@ function getNodeRuntimeEnv() {
  * @param {any} def The default value.
  * @private
  */
-function defaults(obj, def) {
+function defaults(obj, def = { withFaunaMetrics: false }) {
```

Comment:
changing this code would change the behavior of all users of this utility function.


So instead of bolting this in here how about you just use the defaults method inside _execute.

Like:

```
options = util.defaults(options, {withFaunaMetrics: false})
```
this would be the same pattern as query is doing here:

https://github.com/fauna/faunadb-js/blob/ee6f53f9c985bde41976743530e3846dee058587/src/Client.js#L285

Reply:
It looks like the client default values are actually here https://github.com/fauna/faunadb-js/pull/625/files/db7057a186f80944e548476736b361dc613beb66#diff-b880f47f381d68dcd7473fb094acb874cd50b1140cdf2bfd7a368fa509738073L172

Reply:
I updated them to include the `metrics` default